core/icmp-jna/src/main/java/org/opennms/jicmp/standalone/PingReply.java

```diff
@@ -33,6 +33,9 @@
 /**
  * PingReply
  *
+ * @deprecated This interface has a lot of overlap with {@link EchoPacket}...
+ * one of them needs to be moved into an API project.
+ * 
  * @author brozow
  */
 interface PingReply {
```

Comment:
I'm going to defer this because we need to go through all of our code and add the @Deprecated annotation. It's not regularly used, at least in older code.